engine/src/main/java/org/hibernate/search/engine/metadata/impl/TypeMetadata.java

```diff
@@ -504,13 +505,24 @@ public boolean isJpaIdUsedAsDocumentId() {
 		private XProperty jpaProperty;
 		private final Set<SortableFieldMetadata> classBridgeSortableFieldMetadata = new LinkedHashSet<>();
 
-		public Builder(Class<?> indexedType, IndexManagerType indexManagerType, ConfigContext configContext) {
-			this( indexedType, buildScopedAnalyzerReferenceBuilder( indexManagerType, configContext ) );
+		public Builder(Class<?> indexedType, ConfigContext configContext, ParseContext parseContext) {
+			this.indexedType = indexedType;
+			if ( parseContext.skipAnalyzers() ) {
+				this.analyzerReferenceRegistry = null;
+				this.scopedAnalyzerReferenceBuilder = null;
+			}
+			else {
+				IndexManagerType indexManagerType = parseContext.getIndexManagerType();
+				this.analyzerReferenceRegistry = configContext.getAnalyzerReferenceRegistry( indexManagerType );
+				this.scopedAnalyzerReferenceBuilder = new ScopedAnalyzerReference.Builder(
+						analyzerReferenceRegistry.getDefaultAnalyzerReference() );
+			}
 		}
 
-		public Builder(Class<?> indexedType, ScopedAnalyzerReference.Builder scopedAnalyzerReferenceBuilder) {
+		public Builder(Class<?> indexedType, Builder builderToClone) {
```

Comment:
`builderToClone` is misleading here. You don't clone the `Builder`. I think `containerBuilder` would be better.